src/sentry/types/condition_activity.py

```diff
@@ -18,7 +18,7 @@ class ConditionActivityType(Enum):
 
 @dataclass
 class ConditionActivity:
-    group_id: str
+    group_id: int
```

Comment:
does this have other references in the code base where we're sending it a string or no?